### PR TITLE
fixup test runner

### PR DESCRIPTION
**Pull request by turadg**
This one got lost in the commit cleanups:
- run tests by ENTRYPOINT, not RUN

These I'm slipping in as well:
- refactor: consistency of run_ scripts
- chore: disable bash debugging
- refactor: import NonNullish
- feat: dry flag to output build commands
- [refactor: sdkImageTag](https://github.com/Agoric/agoric-3-proposals/pull/8/commits/5203326889564483aeb814699d84e3bd31f2278a)
- [feat: use --link to reduce invalidations](https://github.com/Agoric/agoric-3-proposals/pull/8/commits/5c45ce30c50e27349a7ef5cc452c4e1e5ef512c8)
- [feat: refresh Dockerfile before build](https://github.com/Agoric/agoric-3-proposals/pull/8/commits/54b58764acf22497d3cf86d46ec827c601f705cd)

### CODE REVIEW

File: proposals/43:upgrade-11/legacy.sh [hunk 1]
```diff
@@ -4,7 +4,6 @@
 source /usr/src/upgrade-test-scripts/env_setup.sh
 
 # Enable debugging
-set -x
```
Comment (dckc) [inline proposals/43:upgrade-11/legacy.sh:7]: the "Enable debugging" comment stays? on purpose?
not a big deal...

File: makeDockerfile.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ ENV UPGRADE_TO=${to} THIS_NAME=${agZeroUpgrade}
 # put env functions into shell environment
 RUN echo '. /usr/src/upgrade-test-scripts/env_setup.sh' >> ~/.bashrc
 
-COPY --chmod=755 ./upgrade-test-scripts /usr/src/upgrade-test-scripts
+COPY --link --chmod=755 ./upgrade-test-scripts /usr/src/upgrade-test-scripts
```
Comment (dckc) [inline makeDockerfile.ts:32]: TIL: [COPY --link](https://docs.docker.com/engine/reference/builder/#copy---link)

File: makeDockerfile.ts [hunk 3]
```diff
@@ -171,5 +171,11 @@ for (const proposal of readProposals()) {
...
 }
 
-const contents = blocks.join('\n');
-fs.writeFileSync('Dockerfile', contents);
+export function refreshDockerfile() {
+  const contents = blocks.join('\n');
+  fs.writeFileSync('Dockerfile', contents);
+}
+
+if (require.main === module) {
```
Comment (dckc) [inline makeDockerfile.ts:179]: I thought we used ES modules and node doesn't support this idiom in that context.

Does .ts not get compiled to ES modules?

Comment (turadg) [reply makeDockerfile.ts:179]: Node only uses ESM if `package.json` has `"type": "module"`. But there's no package.json here so it defaults to CJS

File: proposals/43:upgrade-11/legacy.sh (hunk 1 above)
Comment (turadg) [reply proposals/43:upgrade-11/legacy.sh:7]: oops, will remove. (CI's not passing anyway)

**Pull request merged**